cart.html

```diff
@@ -1,90 +1,91 @@
+      <a href="index.html"><img src="img/logo.png" /></a>
 
-            <article class="mt-5"></article>
-        </section>
+      <nav>
+        <ul class="menu">
+          <li class="toggle">
+            <a href="#"><i class="fas fa-bars"></i></a>
+          </li>
+          <li class="item"><a href="index.html">Home</a></li>
+          <li class="item"><a href="shop.html">Shop</a></li>
+          <li class="item"><a href="gallery.html">Gallery</a></li>
+          <li class="item"><a href="contact.html">Contact</a></li>
+        </ul>
+        <!-- Icons for for the user and cart -->
+        <ul class="login-cart">
+          <!--Login-->
+          <li>
+            <a href="login.html"><i class="fas fa-user"></i></a>
+          </li>
+          <!--Cart-->
+          <li>
+            <a href="cart.html"><i class="fas fa-shopping-cart"></i></a>
+          </li>
+        </ul>
+      </nav>
+    </header>
 
-        <footer>
-            <h5>Planet wine</h5>
-            <p>3700 Willingdon Ave, Burnaby, BC V5G 3H2</p>
-            <div class="social">
-                <ul>
-                    <li>
-                        <a href="#"><i class="fab fa-facebook"></i></a>
-                    </li>
-                    <li>
-                        <a href="#"><i class="fab fa-twitter"></i></a>
-                    </li>
-                    <li>
-                        <a href="#"><i class="fab fa-instagram"></i></a>
-                    </li>
-                    <li>
-                        <a href="#"><i class="fab fa-youtube"></i></a>
-                    </li>
-                </ul>
-            </div>
-            <p>&copy; All Right Reserved</p>
-        </footer>
+    <section>
+      <header>
+        <div class="text-center mt-5">
+          <h1 class="text-dark icon-collapse">Your Cart</h1>
+        </div>
+      </header>
 
-        <!-- <script src="scripts/index.js"></script> -->
-        <script src="scripts/index.js"></script>
-        <script src="scripts/cart.js"></script>
-        <script src="scripts/global-toggle.js"></script>
-    </body>
+      <article class="mt-5"></article>
+    </section>
+
+    <footer>
+      <h5>Planet wine</h5>
+      <p>3700 Willingdon Ave, Burnaby, BC V5G 3H2</p>
+      <div class="social">
+        <ul>
+          <li>
+            <a href="#"><i class="fab fa-facebook"></i></a>
+          </li>
+          <li>
+            <a href="#"><i class="fab fa-twitter"></i></a>
+          </li>
+          <li>
+            <a href="#"><i class="fab fa-instagram"></i></a>
+          </li>
+          <li>
+            <a href="#"><i class="fab fa-youtube"></i></a>
+          </li>
+        </ul>
+      </div>
+      <p>&copy; All Right Reserved</p>
+    </footer>
+
+    <!-- <script src="scripts/index.js"></script> -->
+    <script src="scripts/index.js"></script>
+    <script src="scripts/cart.js"></script>
+    <script src="scripts/global-toggle.js"></script>
+  </body>
 </html>
```

Comment:
Seems like you still have it set to space 2 at your bottom right, set it to space 4 then use prettier again.